cmd/status/cmdstatus.go

```diff
@@ -96,7 +96,7 @@ func (r *StatusRunner) runE(cmd *cobra.Command, args []string) error {
 
 	// Based on the inventory template manifest we look up the inventory
 	// from the live state using the inventory client.
-	identifiers, err := invClient.GetClusterObjs(inv)
+	identifiers, err := invClient.GetClusterObjs(inventory.WrapInventoryObj(inv))
```

Comment:
same

Reply:
Updated the code to use the function from provider.InventoryClient.